xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlaySSA.h

```diff
@@ -34,7 +34,6 @@ class CDVDOverlaySSA : public CDVDOverlay
   {
     replace = true;
     m_libass = libass;
-    libass->Acquire();
```

Comment:
I'm suspicious of:
https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleParser.h#L53
which seems to create a new CDVDOverlay and lose track of the original.
Returning ``o`` rather than ``o->Clone()`` also seems to fix the leak.
Do you know why we are cloning the object?